File: backend/Rest API PWII/Models/ViewModels/UserViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Rest_API_PWII.Models.ViewModels
{
    public class UserViewModel
    {
        public string       Id                  { get; set; }
        public string       UserName            { get; set; }
        public string       Tag                 { get; set; }
        public string       Email               { get; set; }
        public string       ProfilePicPath      { get; set; }
        public string       CoverPicPath        { get; set; }
        public DateTime?    BirthDate           { get; set; }
        public int?         FollowerCount       { get; set; }
        public int?         FollowingCount      { get; set; }
    }
}

File: frontend/src/model/UserViewModel.js
class UserViewModel {
    id;
    userName;
    tag; 
    email;
    profilePhotoMediaID;
    birthDate;

    contructor( {id, userName, tag, email, profilePhotoMediaID,birthDate}){
        this.id = id;
        this.userName = userName;
        this.tag = tag;
        this.email = email;
        this.profilePhotoMediaID = profilePhotoMediaID;
        this.birthDate = birthDate;
    }
}

export default UserViewModel;